testsuite/do_tests.sh

```diff
@@ -145,8 +145,14 @@ NEST="nest_serial"
 HAVE_MPI="$(sli -c 'statusdict/have_mpi :: =only')"
 
 if test "${HAVE_MPI}" = "true"; then
-  MPI_LAUNCHER="$(sli -c '1 () () mpirun cst 0 get =only')"
-  MPI_LAUNCHER="$(command -v $MPI_LAUNCHER)"
+    MPI_LAUNCHER="$(sli -c 'statusdict/mpiexec :: =only')"
+    MPI_LAUNCHER_VERSION="$($MPI_LAUNCHER --version | head -n1)"
+    # OpenMPI requires --oversubscribe to allow more processes than available cores
+    if [[ "${MPI_LAUNCHER_VERSION}" =~ "(OpenRTE)" ]]; then
```

Comment:
Another point here on `mpirun` vs `mpiexec`.  When called with `--version`, the former returns `(Open MPI)`, the latter `(OpenRTE)`. At the same time, `man mpiexec` shows a man page stating that the two are equivalent and that uses `mpirun` in all examples. Can we be robust here?

Reply:
I did some checks on my machine and it always returned OpenRTE, but think that's all up to your MPI implementation (and which version CMake happens to find on your machine). I have extended the check to include "Open MPI" in 1eb38cf94.